lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -355,15 +355,43 @@ class DetailsRenderer {
    */
   renderNode(item) {
     const element = this._dom.createElement('span', 'lh-node');
-    if (item.snippet) {
-      element.textContent = item.snippet;
-    }
     if (item.selector) {
       element.title = item.selector;
     }
     if (item.path) element.setAttribute('data-path', item.path);
     if (item.selector) element.setAttribute('data-selector', item.selector);
     if (item.snippet) element.setAttribute('data-snippet', item.snippet);
+    element.innerHTML = `<div style="color: black;font-family: var(--text-font-family);margin-bottom: 10px;">${item.textContent}</div>`;
+    const snippetEl = document.createElement('div');
+    snippetEl.style.fontSize = '12px';
+    snippetEl.innerText = item.snippet;
+    element.appendChild(snippetEl);
+
+    const fullpageScreenshotUrl = __LIGHTHOUSE_JSON__.audits['full-page-screenshot'].details.data;
```

Comment:
What's a good way to access this in the renderer?

Reply:
I would say it should probably be passed into a `DetailsRenderer` as the context of the report, almost like `setTemplateContext`

Reply:
Passing it into the `DetailsRenderer` constructor now.